### PR TITLE
Reduce memory usage

### PR DESCRIPTION
**Pull request by m09**

### CODE REVIEW

File: lookout/style/format/features.py [hunk 1]
```diff
@@ -274,7 +274,7 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
                            (vnode.start.line in file_lines if file_lines is not None else True)])
 
         y = numpy.concatenate(labels)
-        X = numpy.full((y.shape[0], self.count_features(FeatureType.all)), -1)
+        X = numpy.full((y.shape[0], self.count_features(FeatureType.all)), -1, dtype=numpy.int8)
```
Comment (zurk) [inline lookout/style/format/features.py]: But position features can overflow `int8`.

Comment (vmarkovtsev) [reply lookout/style/format/features.py]: Why not uint8 and 256?

Comment (m09) [reply lookout/style/format/features.py]: we have -1 to denote the absence of features

Comment (vmarkovtsev) [reply lookout/style/format/features.py]: Make it `0xff` then. MANY lines will exceed 127

**Comment by vmarkovtsev** [comment]
Waiting for uint8 here

File: lookout/style/format/features.py [hunk 2]
```diff
@@ -274,7 +274,7 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
                            (vnode.start.line in file_lines if file_lines is not None else True)])
 
         y = numpy.concatenate(labels)
-        X = numpy.full((y.shape[0], self.count_features(FeatureType.all)), -1)
+        X = numpy.zeros((y.shape[0], self.count_features(FeatureType.all)), dtype=numpy.uint8)
```
[the remark this answers is not in the document]
Comment (m09) [reply lookout/style/format/features.py]: It does work, in combination with sparse matrices.

[the remark this answers is not in the document]
Comment (zurk) [reply lookout/style/format/features.py]: ok, cool then.

**Pull request merged**